spec/requests/organisation_relationships_controller_spec.rb

```diff
@@ -0,0 +1,473 @@
+require "rails_helper"
```

Comment:
thanks for making this, definitely a better approach than cramming it into the org controller spec